### PR TITLE
fix(chaosctl): best-effort-chaosctl

### PR DESCRIPTION
**Pull request by STRRL**
Signed-off-by: STRRL <str_ruiling@outlook.com>

### What problem does this PR solve?

Fix https://github.com/chaos-mesh/chaos-mesh/issues/1400
### What is changed and how does it work?

do not quit the loop while debugPod returns an error.

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Review by Yiyiyimu — APPROVED** [review]
Yeah I didn't consider the situation when mode=one. Thanks for the fix!

LGTM by now

**Review by cwen0 — APPROVED** [review]
LGTM

**Comment by zhouqiang-cl** [comment]
@STRRL Please resolv conflicts

File: pkg/chaosctl/debug/iochaos/iochaos.go [hunk 1]
```diff
@@ -47,12 +47,9 @@ func Debug(ctx context.Context, chaos runtime.Object, c *cm.ClientSet, result *c
...
 		podName := pods[i].Name
 		podResult := cm.PodResult{Name: podName}
-		err := debugEachPod(ctx, pods[i], daemons[i], ioChaos, c, &podResult)
+		_ = debugEachPod(ctx, pods[i], daemons[i], ioChaos, c, &podResult)
 		result.Pods = append(result.Pods, podResult)
-		if err != nil {
-			// TODO: should follow best-effort pattern
-			return errors.Wrap(err, fmt.Sprintf("failed to debug io chaos for pod %s", podName))
-		}
+		// TODO: V(4) log when err != nil, wait for #1433
```
Comment (WangXiangUSTC) [inline pkg/chaosctl/debug/iochaos/iochaos.go:52]: should we delete `wait for #1433` since 1433 is merged

Comment (STRRL) [reply pkg/chaosctl/debug/iochaos/iochaos.go:52]: I will append logging and finish this TODO in another PR. :)

**Pull request merged**